aggregate-snapshot/aggregate-snapshot-repository/src/test/java/uk/gov/justice/services/eventsourcing/jdbc/snapshot/SnapshotRepositoryJdbcIT.java

```diff
@@ -87,7 +88,8 @@ public void shouldRetrieveLatestSnapshotWithCorrectType() {
         jdbcRepository.storeSnapshot(aggregateSnapshot2);
         jdbcRepository.storeSnapshot(aggregateSnapshot3);
 
-        final Optional<AggregateSnapshot<RecordingAggregate>> snapshot = jdbcRepository.getLatestSnapshot(STREAM_ID, TYPE);
+        Poller poller = new Poller();
+        final Optional<AggregateSnapshot<RecordingAggregate>> snapshot  = poller.pollUntilFound(() -> jdbcRepository.getLatestSnapshot(STREAM_ID, TYPE));
```

Comment:
Is this change related to this pull request?

Reply:
flaky test was failing my local build intermittently